iohub/ngff.py

```diff
@@ -900,6 +900,27 @@ def update_channel(self, chan_name: str, target: str, data: ArrayLike):
         ortho_sel[ch_ax] = ch_idx
         img.set_orthogonal_selection(tuple(ortho_sel), data)
 
+    @property
+    def scale(self) -> list[float]:
+        """
+        Helper function for scale transform metadata of
+        highest resolution scale.
+        """
+        transforms = (
+            self.metadata.multiscales[0].datasets[0].coordinate_transformations
+        )
+        scale_transforms = list(
+            filter(lambda x: x.type == "scale", transforms)
+        )
+
+        if len(scale_transforms) != 1:
```

Comment:
Having multiple items here is not ambiguous. The specification is clear about how they should be applied:

> The transformations in the list are applied sequentially and in order.

So we just need to multiply element-wise.